src/GitTabExpansion.ps1

```diff
@@ -180,6 +180,27 @@ function script:expandGitAlias($cmd, $rest) {
     }
 }
 
+function script:expandParams($cmd, $filter) {
+    $params[$cmd] -split ' ' |
+        where { $_ -like "$filter*" } |
+        sort |

```

Comment:
Please replace all aliases with the full command name: where -> where-object, sort -> sort-object, foreach -> foreach-object.  We have folks running this on PowerShell Core on Linux and sort is a native command (not an alias for Sort-Object).